config/install/field.storage.node.field_homeworld.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_homeworld
+field_name: field_homeworld
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node_type
```

Comment:
This needs to reference "Content" instead of "Content Type" in the field settings.
<img width="416" alt="people_homeworld_field_storage" src="https://github.com/notS0Solo/swapimigration/assets/1434979/cd0b3692-342a-48ba-a5a7-3d437b7dce1a">
Once that change is made, update the field configuration to reference the "Planet" content type as well.

Reply:
Done and ready for review.